drivers/periph_common/Kconfig.spi

```diff
@@ -20,6 +20,14 @@ config MODULE_PERIPH_SPI_RECONFIGURE
     bool "Pin reconfiguration support"
     depends on HAS_PERIPH_SPI_RECONFIGURE
 
+config MODULE_PERIPH_SPI_ON_QSPI
+    bool "Run SPI mode over QSPI peripherial"
+    depends on HAS_PERIPH_SPI_ON_QSPI
+
+config MODULE_PERIPH_INIT_SPI_ON_QSPI
```

Comment:
This does not exist

Reply:
I think it cannot be grepped for but is automatically created.  It shows up in `BOARD=same54-xpro make clean info-modules -C tests/periph_spi | grep periph_init_spi_on_qspi`.

Reply:
Makefile generates this things, better add it to this list: https://github.com/RIOT-OS/RIOT/blob/0de8bfaadc72ae0d45db2070de54dc8d92ac5e22/makefiles/features_modules.inc.mk#L14-L21